### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/06/day5.html?fbclid=IwAR2k41hUyv93kYq1VXUL8Pa4fA5KXByp81viGRXmfAXKx7OeKl_RqDlnqNw_aem_AYwMwjo4H-PDJ-uTx82L_lJ4TxQXtF_FIftDeYfZWT12MGsJmT9i0VccN2_i2iquMXE

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#14] [Comments]
# https://mnfienen.github.io/talulat/2023/08/06/day5.html?fbclid=IwAR2k41hUyv93kYq1VXUL8Pa4fA5KXByp81viGRXmfAXKx7OeKl_RqDlnqNw_aem_AYwMwjo4H-PDJ-uTx82L_lJ4TxQXtF_FIftDeYfZWT12MGsJmT9i0VccN2_i2iquMXE

Day 5 - Somedays, you get a headwind all day miles today: 76 total miles: 279 km today: 122 total km: 368

https://mnfienen.github.io/talulat/2023/08/06/day5.html?fbclid=IwAR2k41hUyv93kYq1VXUL8Pa4fA5KXByp81viGRXmfAXKx7OeKl_RqDlnqNw_aem_AYwMwjo4H-PDJ-uTx82L_lJ4TxQXtF_FIftDeYfZWT12MGsJmT9i0VccN2_i2iquMXE

**Comment by SmittysGitHub** [comment]
Nice work!  Good advice from your New Zealand friend!  Headwinds are definitely a challenge!!
